SUMMARY.md

```diff
@@ -3,11 +3,21 @@
 * [Introduction](introduction.md)
   * [Problem description](introduction/problem-description.md)
   * [Research questions](introduction/research-questions.md)
-* [Existing communities and software development](existing-communities/README.md)
-  * [Current efforts](existing-communities/efforts.md)
-* [The Software Development Process](software-development/README.md)
-  * [Limiting factors](software-development/limitations.md)
-  * [Improvements](software-development/Improvements.md)
-* [The software development ecosystem](ecosystem/README.md)
-* [Conclusion](conclusion.md)
+  * [Report Structure](introduction/report-structure.md)
+* [Communities](communities.md)
+  * [Ecosystems](communities/ecosystems.md)
+  * [Intentional communities](communities/intentional-communities.md)
+  * [MMO-RPG communities](communities/mmo-rpg-communities.md)
+  * [Software Communities](communities/software-communities.md)
+* [Software lifecycle](software-lifecycle.md)
```

Comment:
I am reevaluating what this chapter should be called. Either `Software lifecycle` or `Requirements Engineering`. It doesn't really matter for this PR, but you should know